webapp/src/pages/ResponseList/components/ResponseListComponent.js

```diff
@@ -0,0 +1,494 @@
+            accessor: u => u.start_date,
+            minWidth: 150
+          },
+          {
+            id: "is_submitted",
+            Header: <div className="response-submitted">{t("Submitted")}</div>,
+            accessor: u => u.is_submitted ? "True" : "False",
+            minWidth: 80
+          },
+          {
+            id: "is_withdrawn",
+            Header: <div className="response-withdrawn">{t("Withdrawn")}</div>,
+            accessor: u => u.is_withdrawn ? "True" : "False",
+            minWidth: 80
+          },
+          {
+            id: "submitted_date",
+            Header: <div className="response-submitted-date">{t("Submitted Date")}</div>,
+            accessor: u => u.submitted_date,
+            minWidth: 150
+          },
+          {
+            id: "reviewers",
+            Header: <div className="response-reviewers">{t("Reviewers")}</div>,
+            accessor: u => u.reviewers.map(r => r.reviewer_name).join("; "),
+            minWidth: 300
+          }
+        ];
+
+        return columns;
+    }
+
+    toggleUnsubmitted = () => {
+        this.setState({
+            includeUnsubmitted: !this.state.includeUnsubmitted
+        }, this.getResponseList);
+    }
+
+    handleChange = event => {
+        const value = event.target.value;
+        this.setState({ 
+            newReviewerEmail: value,
+            reviewerAssignError: "",
+            reviewerAssignSuccess: ""
+        });
+    };
+
+    assignReviewer = () => {
+        reviewService.assignResponsesToReviewer(this.props.event.id, this.state.selectedResponseIds, this.state.newReviewerEmail)
+            .then(response => {
+                this.setState({
+                    reviewerAssignError: response.error,
+                    newReviewerEmail: response.error ? this.state.newReviewerEmail : "",
+                    numReviewsAssigned: response.error ? 0 : this.state.selectedResponseIds.length,
+                    assignedReviewerEmail: response.error ? "" : this.state.newReviewerEmail,
+                    reviewerAssignSuccess: !response.error
+                });
+                if (!response.error) {
+                    this.refreshResponses();
+                }
+            });
+    }
+
+    removeTag = (response, tag) => {
+        this.setState({
+          selectedResponse: response,
+          selectedTag: tag,
+          confirmRemoveTagVisible: true
+        });
+    }
+    
+    addTag = (response) => {
+        const tagIds = response.tags.map(t=>t.id);
+        this.setState({
+          selectedResponse: response,
+          tagSelectorVisible: true,
+          filteredTags: this.state.tags.filter(t=>!tagIds.includes(t.id))
+        })
+    }
+
+    onSelectTag = (tag) => {
+        responsesService.tagResponse(this.state.selectedResponse.response_id, tag.id, this.props.event.id)
+        .then(resp => {
+            if (resp.statusCode === 201) {
```

Comment:
Seems like this service calls it `status` instead of `statusCode`, so this `if` check is not passing and this is why you're not seeing the change in the table after you've added or removed a tag. An unfortunate inconsistency in our services conventions (or lack thereof)! :(

Reply:
By the way, I figured this out by running the debugging in the Sources tab of the Chrome developer tools - not sure if you're using that tool already, but can highly recommend it if you're not!

Reply:
Thanks - fixed! :D